public class MathCompany{
  public void doMath(int x){
    System.out.println("Before bad code");
    try{
      int y = 10 / x;
    }catch(ArithmeticException e){
      System.out.prinltn("doMath: I have caught the ArithmeticException");
    }
    System.out.println("After bad code");
  }
}